src/main/java/frc/robot/AlignPosition.java

```diff
@@ -60,6 +60,31 @@ public static Pose2d getAlignPose() {
             public static void setPosition(AlignPosition alignPos){
                 alignPosition = alignPos;
                 System.out.println("yup and you're banned and youre silly, and youre doxxed, and youre doing too much, and youre... yeah that's enough. Well no its not enough because anyone who has seen this code knows that we can't write two lines of code because NOOOO that would be 2 ez. We have to account for every flippin variable on the field and that everything has to be perfect. and then we have to run it by brandon and then then then... error 404 couldn't find the rest of this rant");
```

Comment:
We should probably go ahead and remove this.

Reply:
removed